src/content/cookbook/plugins/play-video.md

```diff
@@ -85,6 +85,25 @@ The `video_player` plugin can only play asset videos in iOS simulators.
 You must test network-hosted videos on physical iOS devices.
 :::
 
+### MacOS
+
+If you use network-based videos, [add the
+`com.apple.security.network.client`
+entitlement](https://docs.flutter.dev/platform-integration/macos/building#entitlements-and-the-app-sandbox)
```

Comment:
```suggestion
### macOS

If you use network-based videos, 
[add the `com.apple.security.network.client` entitlement][mac-entitlement].

[mac-entitlement]: platform-integration/macos/building#entitlements-and-the-app-sandbox
```

Reply:
TIL that macOS starts with a lower "M" I thought in the other readme that this would be a typo